app/models/jellyfish_demo/service/storage.rb

```diff
@@ -0,0 +1,63 @@
+module JellyfishDemo
+  module Service
+    class Storage < ::Service
+      def actions
+        actions = super.merge :terminate
+        actions
+      end
+
+      def provision
+        server = nil
+
+        handle_errors do
+          # TODO: EACH ANSWER SHOULD BE AN OPTIONAL DETAIL IF THEY EXIST
+          random_id = 1+rand(99999)
+          random_ip = rand(999)
+
+          details = {
+            image_id: self.product.answers.find { |x| x.name == 'image_id' }.value,
```

Comment:
The `Product` and `Service` classes provide a helper `settings` which can be used to quickly get to this data.

`settings[:image_id]` for example.